pkg/logutil/logutil.go

```diff
@@ -30,3 +31,24 @@ func NewWriterToFile(maxSize int, dirPath, name string) (io.Writer, error) {
 	return lw, nil
 
 }
+
+// TeeOutput tees out output to a file path specified by env var `UROOT_LOG_PATH` up to a max limit. Creates necessary directories for the specified logpath if they don't exist.
+func TeeOutput(writer io.Writer, maxSize int) (io.Writer, error) {
+	logPath := os.Getenv("UROOT_LOG_PATH")
+	if logPath != "" {
+		dir := filepath.Dir(logPath)
+		if _, err := os.Stat(dir); os.IsNotExist(err) {
+			log.Printf("Log directory %s doesn't exist, creating...", dir)
+			if err := os.MkdirAll(dir, 0700); err != nil {
+				return nil, err
```

Comment:
Don't you want to at least say something here? You could do a log.Printf or something before returning?